services/apps/data_sink_worker/src/repo/member.repo.ts

```diff
@@ -128,9 +128,6 @@ export default class MemberRepository extends RepositoryBase<MemberRepository> {
         ...data,
         id,
         tenantId,
-        reach: {
-          total: -1,
```

Comment:
isn't this supposed to be 0 by default?

Reply:
It was -1 before, so I think we should keep it like this :)